src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2293,12 +2293,145 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
                         {
                             return;
                         }
+
+                    case '>':
```

Comment:
How about this idea: we stop recognizing `>>>>>>>` as a conflict marker, but continue having `>>>>>>>` end a skipped text section that is started with a `=======`. I think that would remove the compat break and still retain all the value of the feature. Does that work?

---
In reply to: [103549556](https://github.com/dotnet/roslyn/pull/15850#discussion_r103549556) [](ancestors = 103549556)